Fix #1621 Updating obsolete Heroku migration page (#1634)
Scalingo/documentation: src/_posts/platform/getting-started/2000-01-01-how-to-migrate-from-heroku.md

> ---

Reviewer: This page is called from two places:
https://github.com/Scalingo/documentation/blob/5ac1f04e478079b4bdddfd60dff827beb546d9fd/_config.yml#L55

and

https://github.com/Scalingo/documentation/blob/5ac1f04e478079b4bdddfd60dff827beb546d9fd/redirections.yml#L161-L163

The redirection can be deleted. For the `_config.yml`, we may want to ask the marketing as it's currently highlighted on the doc index:

![2022-05-02_19-50](https://user-images.githubusercontent.com/1567651/166298790-de9cc61f-46dd-4911-8f76-98efd1dae461.png)

Can you ask marketing what is their thought about this?

Moreover, deleting this page means we should clearly deprecate this tool (https://github.com/Scalingo/heroku2scalingo). You should open a PR to mark it as deprecated, then ask Léo to archive the repository. You can take inspiration from this PR: https://github.com/Scalingo/puppeteer-buildpack/pull/1

index: 11
---

> Scalingo is a Platform as a Service [highly compatible with Heroku]({% post_url platform/getting-started/2000-01-01-heroku-compatibility %}). This page is here to help your migrate from Heroku to Scalingo as easir as possible.

Reviewer: ```suggestion
Scalingo is a Platform as a Service [highly compatible with Heroku]({% post_url platform/getting-started/2000-01-01-heroku-compatibility %}). This page is here to help your migrate from Heroku to Scalingo as easily as possible.
```

## Requirements

To be able to follow this tutorial, we assume that you have:

- A Scalingo account
- The Scalingo [CLI]({% post_url platform/cli/2000-01-01-start %}) installed
- A local copy of the repository containing your app’s code

Please ensure you are running all commands above from the root of the repository containing your app’s code. 

```bash
$ cd <app directory>

> ```

Reviewer: Not sure this code block is really useful...

Reviewer 2: I wondered the same, but I like that it state "every" command. Can be useful for beginners.

Note: All operations described below can also be done through our web dashboard (instead of the command line tool). 

## Migration Steps

### Creating your Scalingo app

> - Start by login into the Scalingo platform using our `cli` :

Reviewer: Not sure about this one but it seems to be a better phrasing, WDYT?

```suggestion
- Start by logging in to the Scalingo platform using our `cli` :
```

```bash
$ scalingo login
```

> - You can now create your app : 

Reviewer: There is no space before a colon

```suggestion
- You can now create your app:
```

```bash

> # Replace <my-app> by your actual app name
> $ scalingo create <my-app>

Reviewer: In all the documentation we use `my-app` without the `<` and `>` characters. Hence I would keep this page consistent with the remaining of the doc pages:

```suggestion
# Replace my-app by your actual app name
$ scalingo create my-app
```

Reviewer 2: Oups that's my doing

```

### Configure Environment Variables

> - First retrieve the environment variables from your current Heroku app context :

Reviewer: ```suggestion
- First retrieve the environment variables from your current Heroku app context:
```

```bash
$ heroku config
```

- Those environment variables need to be declared in your Scalingo app context. For instance, this command will set the `NODE_ENV` environment variable:  

```bash

> # Replace <my-app> by your actual app name
> $ scalingo -a <my-app> env-set NODE_ENV=production

Reviewer: - I would remove the comment as you already explained that in the previous code block
- Documentation should use long option name (`--app`) instead of the short one (`-a`)
- I would keep consistency with other doc pages and use `my-app` rather than `<my-app>`

```suggestion
$ scalingo --app my-app env-set NODE_ENV=production
```

```

> More informations about environnements variables can be found in the [dedicated documentation page]({% post_url platform/app/2000-01-01-environment %}).

Reviewer: "information" is uncountable in English.

```suggestion
More information about environnement variables can be found in the [dedicated documentation page]({% post_url platform/app/2000-01-01-environment %}).
```

> ### Deploying your app 

Reviewer: ```suggestion
### Deploying Your App
```

> - You are now ready to deploy your app on Scalingo :  

Reviewer: ```suggestion
- You are now ready to deploy your app on Scalingo:
```

```bash
$ git push scalingo master
```

> ### Database migration

Reviewer: ```suggestion
### Database Migration
```

> - Once you have set your Scalingo addons according to those you had on Heroku, you will need to migrate your database by dumping it from Heroku and restoring it to Scalingo. 

Reviewer: Not sure a future tense is required here
```suggestion
- Once you have set your Scalingo addons according to those you had on Heroku, you need to migrate your database by dumping it from Heroku and restoring it to Scalingo. 
```

  * [Dump and restore a MongoDB database]({% post_url databases/mongodb/2000-01-01-dump-restore %})
  * [Dump and restore a PostgreSQL database]({% post_url databases/postgresql/2000-01-01-dump-restore %})
  * [Dump and restore a MySQL database]({% post_url databases/mysql/2000-01-01-dump-restore %})

> ### Where to go next

Reviewer: ```suggestion
### Where to Go Next?
```

> - If your app is using a custom domain, follow the [instruction]({% post_url platform/app/2000-01-01-domain %}) to update your DNS configuration.

Reviewer: ```suggestion
- If your app is using a custom domain, follow the [instructions]({% post_url platform/app/2000-01-01-domain %}) to update your DNS configuration.
```

- Managing your [SSL certificate]({% post_url platform/app/2000-01-01-ssl %}).  

> ### Need some help? 

Reviewer: ```suggestion
### Need Some Help? 
```

> Have questions or need to report an issue? Feel free to reach our support team for further assistance through our live chat or by sending an email to [support@scalingo.com](mailto:support@scalingo.com)

Reviewer: ```suggestion
Have questions or need to report an issue? Feel free to reach our support team for further assistance through our live chat or by sending an email to [support@scalingo.com](mailto:support@scalingo.com).
```
